src/components/SkipNavigation/SkipNavigation.tsx

```diff
@@ -37,10 +37,15 @@ export const SkipNavigation = chakra(
         >
           <List inline noStyling type="ul">
             <li>
-              <Link href={target}>Skip to Main Content</Link>
+              <Link href={target} isUnderlined={false}>
```

Comment:
I still see these as underlined:
<img width="348" alt="Screen Shot 2023-08-30 at 2 50 42 PM" src="https://github.com/NYPL/nypl-design-system/assets/1280564/ef0c45bf-1ee2-428e-865d-f207abfab147">

Reply:
Made this comment before reading the description! ignore.